flow/src/main/scala/org/alephium/flow/io/DatabaseVersion.scala

```diff
@@ -21,10 +21,12 @@ import akka.util.ByteString
 import org.alephium.serde.Serde
 import org.alephium.util.Bytes
 
-final case class DatabaseVersion(value: Int) extends AnyVal
+final case class DatabaseVersion(value: Int) extends AnyVal with Ordered[DatabaseVersion] {
+  override def compare(that: DatabaseVersion): Int = value.compare(that.value)
+}
 
 object DatabaseVersion {
   implicit val serde: Serde[DatabaseVersion] = Serde.forProduct1(apply, _.value)
 
-  val currentDBVersion: DatabaseVersion = DatabaseVersion(Bytes.toIntUnsafe(ByteString(0, 1, 0, 0)))
+  val currentDBVersion: DatabaseVersion = DatabaseVersion(Bytes.toIntUnsafe(ByteString(0, 1, 1, 0)))
```

Comment:
we need to think how to merge and release that, as we will need to ask people to wipe out their db I guess?

Reply:
user can upgrade without any changes, but the downgrade needs to modify the database, we can provide a python script to do this.